### ISSUE TITLE
[BUG]: ios 빌드 에러

### ISSUE THREAD

**Issue by mingyeom1** [mym0404/react-native-naver-map#62]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Version of @mj-studio/react-native-naver-map

2.0.0-alpha.2

### Version of react-native

0.72.6

### What os are you seeing the problem on?

iOS

### What device types are you seeing the problem on?

Simulator, Physcial Device

### What architecture types of react native are you seeing the problem on?

New Architecture(Fabric)

### Expo App

- [ ] I am using Expo

### What happened?

2.0.0-alpha.2 버전에서 ios 빌드 시 에러 발생하는데 확인 부탁드립니다. 

![스크린샷 2024-06-17 오후 4 28 37](https://github.com/mym0404/react-native-naver-map/assets/72376761/4f6f219f-5513-4c5f-b7ad-bf66080774d2)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by mym0404** [comment]
c++20 문법인 `concept`를 인식하지 못하는 것 같습니다. Fabric을 사용중인지, 프로젝트가 C++20 이상으로 빌드되고 있는지, Pod에서 naver map Pod이 C++20으로 빌드가 되고있는지 확인해보셔야 할 것 같습니다.

문제가 지속된다면 1.x 버전을 사용하시거나 RN 버전 업그레이드 후 시도해주세요.

**Comment by junleejaehyuk** [comment]
안녕하세요 Fabric 사용을 안했을때 2.x 버전이 지원될 예정이 있나요?

Fabric으로 인한 타 라이브러리들이 문제를 일으키는경우가 많아서요.!

좋은 패키지에 사용 기여에 감사드립니다. @mym0404

**Comment by mym0404** [comment]
2.x 버전은 Fabric만 지원하게 설계되었습니다. 지원 예정은 없습니다.

**Comment by mingyeom1** [comment]
그럼 혹시 onTapClusterLeaf prop을 1.x버전에서 사용할 수 있게 업데이트 가능할까요? @mym0404

**Comment by mingyeom1** [comment]
![스크린샷 2024-07-02 오후 5 15 45](https://github.com/mym0404/react-native-naver-map/assets/72376761/65efa954-1e9a-488b-95a4-784c1b7c86c9)
Fabric 설정 후 빌드해봐도 해당 에러는 동일하게 발생합니다.

**Comment by mingyeom1** [comment]
> 현재 제 코드에선 재현이 되지 않습니다. New Architecture가 적용된 Reproducing Sample을 첨부한 새로운 이슈를 열어주시면 살펴보겠습니다. 이 이슈는 Fabric 빌드와 관련없는 이슈이기 때문에 닫겠습니다.

넵 새로운 이슈 열겠습니다. 혹시 onTapClusterLeaf prop을 1.x버전 지원 계획은 없으신가요?

**Comment by mym0404** [comment]
네 없을것같습니다. 이미 v2를 위해 코드를 수정을 많이한 상태에서 추가한 기능이라 구조가 변경하기에 어렵습니다.

**Comment by ride-lupin** [comment]
> > 현재 제 코드에선 재현이 되지 않습니다. New Architecture가 적용된 Reproducing Sample을 첨부한 새로운 이슈를 열어주시면 살펴보겠습니다. 이 이슈는 Fabric 빌드와 관련없는 이슈이기 때문에 닫겠습니다.
> 
> 넵 새로운 이슈 열겠습니다. 혹시 onTapClusterLeaf prop을 1.x버전 지원 계획은 없으신가요?

해당 이슈 

> ![스크린샷 2024-07-02 오후 5 15 45](https://private-user-images.githubusercontent.com/72376761/344936302-65efa954-1e9a-488b-95a4-784c1b7c86c9.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjA3NjAyOTksIm5iZiI6MTcyMDc1OTk5OSwicGF0aCI6Ii83MjM3Njc2MS8zNDQ5MzYzMDItNjVlZmE5NTQtMWU5YS00ODhiLTk1YTQtNzg0YzFiN2M4NmM5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MTIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzEyVDA0NTMxOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTEzY2VjNGQzMWM2ZGI0M2E3MjJiNTUwYmQyY2Q2NWM2YTY1YjQ4NjI3ZDc0YzM4ODY3MjJlMWYzMjFiYmM1NDYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.FhNlTD1t2Vt9itpko1j1bNpd7AgG_bNmHlh0oRGzpbc) Fabric 설정 후 빌드해봐도 해당 에러는 동일하게 발생합니다.

해당 에러 해결하셨나요 ?

동일한 에러가 발생하고 있습니다.

**Comment by mym0404** [comment]
재현가능한 새 리액트네이티브 프로젝트를 요청드립니다
